EarlyBird/CoursesTableViewController.swift

```diff
@@ -39,21 +35,17 @@ class CoursesTableViewController: UITableViewController {
         }
         
         let urlRequest = URLRequest(url: url)
-        // set up the session
         let config = URLSessionConfiguration.default
         let session = URLSession(configuration: config)
         
         // make the GET request
         let task = session.dataTask(with: urlRequest) { (data, response, error) in
-            // check for any errors
             guard error == nil else {
                 print("error calling GET on /courses/1")
```

Comment:
I'm seeing a few very detailed error messages, but these are explicitly printed - is there a way to log the errors rather than print them? I don't imagine golf course users of your final product care much that the GET request didn't return data! It might be a good idea to have a generic error message printed to users which describes the scope of that particular error (which app features are affected)